### PR TITLE
feat: added share music modal

### PR DESCRIPTION
**Pull request by nolostra**
Link ref => [#67](https://github.com/code100x/muzer/issues/67)

added a modal that helps u to share music on differnt platforms 

image

### CODE REVIEW

File: app/components/StreamView.tsx [hunk 1]
```diff
@@ -141,31 +149,31 @@ export default function StreamView({
...
-        autoClose: 3000,
-        hideProgressBar: false,
-        closeOnClick: true,
-        pauseOnHover: true,
-        draggable: true,
-        progress: undefined,
-      })
-    })
-  }
+//   const handleShare = () => {
```
[the remark this answers is not in the document]
Comment (nolostra) [reply app/components/StreamView.tsx:152]: that functionality was moved inside of modal , so when a user click on the respected button it moves to respected apps

[the remark this answers is not in the document]
Comment (SujithThirumalaisamy) [reply app/components/StreamView.tsx:152]: You should remove it. If it is moved

[the remark this answers is not in the document]
Comment (nolostra) [reply app/components/StreamView.tsx]: yeah removed

**Comment by SujithThirumalaisamy** [comment]
can we please have a dropdown for the share button sir?

**Comment by SujithThirumalaisamy** [comment]
Fixed on #67

**Pull request closed without merging**